packages/app/package.json

```diff
@@ -126,59 +125,55 @@
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.18.6",
     "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.19.1",
-    "@babel/preset-react": "^7.18.6",
-    "@babel/preset-typescript": "^7.18.6",
-    "@contentful/rich-text-react-renderer": "^15.12.1",
-    "@contentful/rich-text-types": "^15.12.1",
-    "@electron/remote": "^2.0.8",
+    "@babel/preset-env": "^7.16.5",
```

Comment:
why was this downgraded?

Reply:
https://github.com/mudita/mudita-center/pull/1550/commits/c91d78ac7c8639525a774875d2894e91d00f0450